<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterForm</class>
 <widget class="QWidget" name="PrinterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="infoEdit">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>160</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Printer information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Printer name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>100</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Printer code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeEdit">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>60</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Component list</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Printer list</string>
   </property>
  </widget>
  <widget class="QTableView" name="printerTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="componentTable">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reloadButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload form</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPrinterButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePrinterButton">
   <property name="geometry">
    <rect>
     <x>598</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCompButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateCompButton">
   <property name="geometry">
    <rect>
     <x>608</x>
     <y>320</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCompButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Printer id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="printerIdEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
